bigquery/test/tables.test.js

```diff
@@ -418,148 +394,101 @@ describe('bigquery:tables', function () {
+      assert.deepEqual(program.listTables.firstCall.args.slice(0, -1), [datasetId]);
     });
 
     it('should call browseRows', function () {
       var program = getSample().program;
       program.browseRows = sinon.stub();
 
-      program.main(['browse', dataset, table]);
+      program.main(['browse', datasetId, tableId]);
       assert.equal(program.browseRows.calledOnce, true);
-      assert.deepEqual(program.browseRows.firstCall.args.slice(0, -1), [dataset, table]);
+      assert.deepEqual(program.browseRows.firstCall.args.slice(0, -1), [datasetId, tableId]);
     });
 
     it('should call deleteTable', function () {
       var program = getSample().program;
       program.deleteTable = sinon.stub();
 
-      program.main(['delete', dataset, table]);
+      program.main(['delete', datasetId, tableId]);
       assert.equal(program.deleteTable.calledOnce, true);
-      assert.deepEqual(program.deleteTable.firstCall.args.slice(0, -1), [{ dataset: dataset, table: table }]);
+      assert.deepEqual(program.deleteTable.firstCall.args.slice(0, -1), [datasetId, tableId]);
     });
 
-    it('should call importFile', function () {
+    it('should call importLocalFile', function () {
       var program = getSample().program;
-      program.importFile = sinon.stub();
-
-      program.main(['import', dataset, table, file]);
-      assert.equal(program.importFile.calledOnce, true);
-      assert.deepEqual(program.importFile.firstCall.args.slice(0, -1), [{
-        dataset: dataset,
-        table: table,
-        file: file,
-        bucket: undefined,
-        format: undefined
-      }]);
+      program.importLocalFile = sinon.stub();
+
+      program.main(['import', datasetId, tableId, fileName]);
+      assert.equal(program.importLocalFile.calledOnce, true);
+      assert.deepEqual(program.importLocalFile.firstCall.args.slice(0, -1), [datasetId, tableId, fileName]);
+    });
+
+    it('should call importFileFromGCS', function () {
+      var program = getSample().program;
+      program.importFileFromGCS = sinon.stub();
+
+      program.main(['import', datasetId, tableId, fileName, '-b', bucketName]);
+      assert.equal(program.importFileFromGCS.calledOnce, true);
+      assert.deepEqual(program.importFileFromGCS.firstCall.args.slice(0, -1), [datasetId, tableId, bucketName, fileName]);
     });
 
     it('should call copyTable', function () {
       var program = getSample().program;
       program.copyTable = sinon.stub();
 
-      program.main(['copy', srcDataset, srcTable, destDataset, destTable]);
+      program.main(['copy', srcDatasetId, srcTableId, destDatasetId, destTableId]);
       assert.equal(program.copyTable.calledOnce, true);
-      assert.deepEqual(program.copyTable.firstCall.args.slice(0, -1),
-        [srcDataset, srcTable, destDataset, destTable]
-      );
+      assert.deepEqual(program.copyTable.firstCall.args.slice(0, -1), [srcDatasetId, srcTableId, destDatasetId, destTableId]);
```

Comment:
Perhaps in a separate PR we can consider switching from Semistandard to ESLint + JSCS to give us some configurability and better enforceability on our style?